ibm/service/cis/resource_ibm_cis_firewall_rules.go

```diff
@@ -181,7 +185,8 @@ func ResourceIBMCISFirewallrulesUpdate(context context.Context, d *schema.Resour
 	if d.HasChange(cisFilterID) ||
 		d.HasChange(cisFirewallrulesAction) ||
 		d.HasChange(cisFirewallrulesPaused) ||
-		d.HasChange(cisFilterDescription) {
+		d.HasChange(cisFilterDescription) ||
+		d.HasChange(cisFirewallrulesPriority) {
 
```

Comment:
in update handle the support for change in priority

Reply:
Added